### DISCUSSION TITLE
Old Matrix Checks Still There

### DISCUSSION THREAD

**Discussion by jamesward** [community/community#51499] [Actions] [answered]
### Select Topic Area

Bug

### Body

My Actions used to have a matrix which I removed but they are still listed on new PRs but stuck on "Expected — Waiting for status to be reported"  Example PR: https://github.com/GoogleCloudPlatform/kotlin-samples/pull/259

![image](https://user-images.githubusercontent.com/65043/228872624-63611e3c-3658-43b0-9141-3f33cd905522.png)

I've tried removing the Action workflow which didn't fix it.  Seems like a bug.  Any ideas?

**Comment by airtower-luna** [comment, marked as answer]
That looks like there's a branch protection rule that requires the matrix jobs before merging. If you remove them from the workflow, you'll need to remove them from the branch protection rule as well to make it work again.

**Reply by jamesward** [reply]
Oh! That was it.  Thank you!
